stateful-map-test/src/main/java/com/hazelcast/jet/tests/stateful/StatefulMapTest.java

```diff
@@ -198,34 +248,53 @@ private Pipeline buildPipeline(String txMapName, String timeoutMapName) {
                              return null;
                          }
                  );
+
         streamStage
                 .filter(e -> e.getKey() < 0)
-                .writeTo(Sinks.remoteMapWithMerging(
-                        timeoutMapName,
-                        stableClusterClientConfig,
-                        (oldValue, newValue) -> {
-                            if (oldValue != null && !oldValue.equals(newValue)) {
-                                System.out.println("StatefulMapTest timeoutMap duplicate old: "
-                                        + oldValue + ", new: " + newValue);
-                            }
-                            return newValue;
-                        }));
-
+                .writeTo(KafkaSinks.kafka(kafkaPropertiesForSink(), TX_TIMEOUT_TOPIC_PREFIX + clusterName));
         streamStage
                 .filter(e -> e.getKey() >= 0)
-                .writeTo(Sinks.remoteMapWithMerging(
-                        txMapName,
-                        stableClusterClientConfig,
-                        (oldValue, newValue) -> {
-                            if (oldValue != null && !oldValue.equals(newValue)) {
-                                System.out.println("StatefulMapTest txMap duplicate old: "
-                                        + oldValue + ", new: " + newValue);
-                            }
-                            return newValue;
-                        }));
+                .writeTo(KafkaSinks.kafka(kafkaPropertiesForSink(), TX_TOPIC_PREFIX + clusterName));
         return p;
     }
 
+    private Properties kafkaPropertiesForSource() {
+        Properties props = new Properties();
+        props.setProperty("bootstrap.servers", brokerUri);
+        props.setProperty("group.id", UuidUtil.newUnsecureUuidString());
+        props.setProperty("key.deserializer", StringDeserializer.class.getName());
+        props.setProperty("value.deserializer", LongDeserializer.class.getName());
+        props.setProperty("auto.offset.reset", "earliest");
+        props.setProperty("max.poll.records", "32768");
+        return props;
+    }
+
+    private Properties kafkaPropertiesForSink() {
+        Properties props = new Properties();
+        props.setProperty("bootstrap.servers", brokerUri);
+        props.setProperty("group.id", UuidUtil.newUnsecureUuidString());
+        props.setProperty("key.deserializer", LongDeserializer.class.getName());
```

Comment:
we don't need deserializer/auto.offset.reset/max.poll.records properties for sink I guess